src/Components/Scenes/RequestAssets/RequestAsset/RequestAsset.js

```diff
@@ -1,21 +1,57 @@
 import React from 'react';
 import RequestAssetTableContainer from '../RequestAssetTable/RequestAssetTable-Container';
 import RequestAssetJustificationContainer from '../RequestAssetJustification/RequestAssetJustification-Container';
+import AppBarContainer from '../../Chrome/AppBar/AppBar-Container';
```

Comment:
Should we be putting and re-rendering a separate `<AppBarContainer>` on every view? Will also address this at a top level on the MR, but I feel like this could be a good use case for the page wrapper

Reply:
I responded this question. see above comment